build.sh

```diff
@@ -5,11 +5,24 @@ version=$1
 commit=${TRAVIS_COMMIT:-`git rev-parse --verify HEAD`}
 echo "Version: ${version:=`date -u +local-%Y%m%d-%H%M%S`}"
 
-for os in linux darwin windows; do
+
+for os in linux; do  # add darwin and windows in the future
     export GOOS=$os
+    go clean
+    go install github.com/Azure/adx-automation-agent/common
+    go install github.com/Azure/adx-automation-agent/models
+    go install github.com/Azure/adx-automation-agent/httputils
+    go install github.com/Azure/adx-automation-agent/kubeutils
+
     mkdir -p $root/bin/$os
     echo -n "Building droid for $os ..."
```

Comment:
>Building droid for $os ... [](start = 13, length = 26)

Are you planning on using a package manager for dependencies?

Reply:
Properly yes.

Reply:
I recommend dep
https://github.com/golang/dep
Just to not get surprises with breaking changes